tool/internal_pkg/generator/generator.go

```diff
@@ -133,6 +134,11 @@ type Config struct {
 
 	TemplateDir string
 
+	// subcommand template
+	InitOutputDir string
+	TplDir        string
```

Comment:
TplDir and TemplateDir are kinda confusing. We need to think about another idea name.